src/url-params.ts

```diff
@@ -16,11 +17,12 @@ const deviceIdSeparator = /\s*,\s*/;
 export class AnalyticsUrlParams {
 	private deviceIds: Set<string> = new Set();
 	private passedDeviceId: string | undefined;
+	private sessionId: number | null = null;
```

Comment:
As we did with `passedDeviceId`, shall we keep the types `number | undefined`? Any particular reason to assign it to `null`?

Reply:
No reason :) Since the allowed type of `passedDeviceId` has changed to `undefined` we can do the same with `sessionId`.